3_javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,82 @@
+//Find and count the Danish letters
+
+
+function countDanishLetters(inputString) {
+    
+    const letterCount = {
+      total: 0,
+      æ: 0,
+      ø: 0,
+      å: 0
+    };
+  
+    
+    const lowercaseString = inputString.toLowerCase();
+  
+   
+    for (let i = 0; i < lowercaseString.length; i++) {
+     
+      if (lowercaseString[i] === 'æ') {
+        letterCount.æ++;
+        letterCount.total++;
+      } else if (lowercaseString[i] === 'ø') {
+        letterCount.ø++;
+        letterCount.total++;
+      } else if (lowercaseString[i] === 'å') {
+        letterCount.å++;
+        letterCount.total++;
+      }
```

Comment:
nice, but maybe you could think of the way how to access letters automatically so you could get rid of `if` statements?